ssh-u2f.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
OK, this is the wrapper for the SSH client to handle the U2F stuff.
I like this a lot!
At first I only saw solutions with copying the challenge to another external command. -> I did not like this!